app/tests/spec/views/sign_in.js

```diff
@@ -204,13 +204,46 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('sync suggestion - signin-mixin', function () {
+
+      it('setShowSyncSuggestion displays sync suggestion message if no migration', function () {
+        initView();
+        relier.set('service', null);
+
+        sinon.stub(view, 'isSyncMigration', function () {
+          return false;
+        });
+        sinon.stub(view, 'isAmoMigration', function () {
+          return false;
+        });
+
+        sinon.spy(view.metrics, 'logEvent');
+
+        return view.render()
+          .then(function () {
+            assert.lengthOf(view.$('#suggest-sync'), 1);
+            assert.equal(view.$('#suggest-sync').html(), 'Looking for Firefox Sync? <a href="https://mozilla.org/firefox/sync">Get started here.</a>');
+            assert.isTrue(TestHelpers.isEventLogged(metrics, 'sync-suggest.visible'), 'enrolled');
+          });
+      });
+
+      it('setShowSyncSuggestion does not display sync suggestion message if there is a relier service', function () {
+        relier.set('service', 'sync');
+
+        return view.render()
+          .then(function () {
+            assert.lengthOf(view.$('#suggest-sync'), 0);
+          });
+      });
+    });
+
     describe('migration', function () {
       it('does not display migration message if no migration', function () {
         initView();
 
         return view.render()
           .then(function () {
-            assert.lengthOf(view.$('.info.nudge'), 0);
+            assert.lengthOf(view.$('.info.nudge.migration'), 0);
```

Comment:
as this PR adds another `info` panel we should update this test to make sure `no migration` info panel shows. 
the `assert.lengthOf(view.$('.info.nudge.migration'), 0);` is probably the most precise test here in case anyone else adds another `.migration` element somewhere. 

You can either use `assert.lengthOf(view.$('.info.nudge.migration'), 0);` or call the new class `.info-migration` and query for that (`.info-migration` would hopefully namespace it enough to make sure that the class is only use in the `info` panel)